plog/plog

```diff
@@ -459,6 +469,28 @@ main () {
     shift $(($OPTIND - 1))
 
 
+# Check compression type
+    if $(command -v  bzip2 >/dev/null) ; then
+        PLOG_COMPRESSED_EXT="${PLOG_COMPRESSED_EXT:-bz2}"
+    else
+        PLOG_COMPRESSED_EXT="${PLOG_COMPRESSED_EXT:-gz}"
+    fi
+
+    if [[ ! "$PLOG_COMPRESSED_EXT" =~ ^(bz2|gz)$ ]]; then
+        echo "PLOG_COMPRESSED_EXT must be bz2 or gz, was $PLOG_COMPRESSED_EXT"
+        exit 1
+     fi
```

Comment:
Maybe this should be in getopts?

Reply:
Or maybe before `getopts`? I put it here so that it occurs after all the manipulation of the variable and it collects all use of the variable (except for `-g`) in this one section.

Reply:
Mmm good point, I guess it's fine